### DISCUSSION TITLE
What are we going to do about new username format?

### DISCUSSION THREAD

**Discussion by nickheyer** [Rapptz/discord.py#9450] [Q&A] [answered]
https://news.ycombinator.com/item?id=35805987

How will this affect existing Discord Py bots currently in prod? 

How can we efficiently change our code to identify (or prepare to identify) users based on the new username format, as opposed to the common and now deprecated <username>#<discrim> format? 

![image](https://github.com/Rapptz/discord.py/assets/60236014/a77c3a96-9c8b-4e4e-8666-0e41a5903f39)

**Comment by MajorTanya** [comment, marked as answer]
The update with support for this went live today, it's Version 2.3.0 of discord.py
